site/en/README.md

```diff
@@ -0,0 +1,30 @@
+# Information Regarding Each Directory
+---
+
+A file to give directions over where each file leads to.
+
+- **about :** Includes information about `Tensor White Papers` involving `Large-Scale Machine Learning on Heterogeneous Distributed Systems`and the white paper `TensorFlow: A System for Large-Scale Machine Learning` with its asbtract.
+
+- **api_docs :** Includes information about the different documentations and pages for APIs that it has for each programming language that Tensorflow currently supports - information is in the `index.md`.
+
+- **community :** Information for the different communities and chatting platforms for the sake of discusssing Tensorflow and related for the purpose of contribution and collaboration. Has a list for `General TensorFlow lists`, `Project-specific lists`, and `Special Interest Groups` exist in the file `forums.md`.
+
+- **datasets :** Contains a single .md file which provides the link to all datasets that can be imported from Tensorflow. A direct link for the aforementioned resource / for the docs is available as this [link](https://github.com/tensorflow/datasets/tree/master/docs).
+
+- **federated :** Guides for TensorFlow Federated ( TFF ), a direct link is [this](https://github.com/tensorflow/federated/tree/master/docs).
```

Comment:
I'm not sure adding all the subsites really adds much, and this file will just go out-of-date without actively maintaining it.

Reply:
So should I make it more concise? Or just drop the ideas as a whole?

Reply:
Let's make it more concise and drop the subsites (anything that links off to another repo). Then we can add a general statement about project-specific docs. That way this file can be a general overview without having to change much. Thanks

Reply:
Alright, I'll look into it! Thank you.